strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/Maven2LayoutProvider.java

```diff
@@ -85,8 +85,7 @@ public boolean isMavenMetadata(RepositoryPath path)
             switch (attributeType)
             {
                 case ARTIFACT:
-                    value = BooleanUtils.isTrue((Boolean) value) && !isMavenMetadata(repositoryPath) &&
```

Comment:
@fuss86 `MavenArtifactUtils.isGAV(repositoryPath)` would be `false` for `maven-metadata.xml`?

Reply:
yes

Reply:
:+1: